.env.sample

```
@@ -0,0 +1,12 @@
+DATABASE_URL = "mysql://password@ipAddress/sql"
+PORT = 3000
+
+TYPEORM_CONNECTION = mysql
+TYPEORM_HOST = ipAddress
+TYPEORM_USERNAME = username
+TYPEORM_PASSWORD = userpassword
+TYPEORM_DATABASE = database
+TYPEORM_PORT = 3306
+TYPEORM_LOGGING = TRUE
+
+TYPEORM_SECRETKEY
```

```suggestion
TYPEORM_SECRETKEY
```
해당 환경변수는 무엇인가요? 또한 변수 선언만 되어있고 값이 할당되어 있지 않은 상태이기 때문에 더더욱 의미가 모호합니다!